packages/server/src/jsRunner/bundles/snippets.ts

```diff
@@ -0,0 +1,19 @@
+// @ts-ignore
+// eslint-disable-next-line local-rules/no-budibase-imports
+import { iifeWrapper } from "@budibase/string-templates/iife"
+
+export default new Proxy(
+  {},
+  {
+    get: function (_, name) {
```

Comment:
Should this not be running in isolated vm? Not sure why another bundle is needed

Reply:
The idea behind these bundles is injecting 3rd party libraries. As far as I understand, the snippets are already js objects that can be injected directly in the isolate. I think it is fine to inject it as you are, but I think this wrapper could be handled on runtime

Reply:
The snippets definitions are stringified and injected in, without a bundle, since they are just primitives. I assumed using a proxy object would require pollyfilling to run in the isolate, which is what happens when this is compiled. I can try injecting this code raw (like I do with the definitions) in case it runs fine. I'd rather leave this as is for now in order to get this released, but it's something I can try after.